extensions/fetch/Cargo.toml

```diff
@@ -24,3 +24,8 @@ serde = { version = "1.0.125", features = ["derive"] }
 tokio = { version = "1.8.0", features = ["full"] }
 tokio-stream = "0.1.5"
 tokio-util = "0.6.7"
+
+# until gets released https://github.com/seanmonstar/reqwest/issues/1210
```

Comment:
Regarding the comment, it is deleted in next commit, the initial idea was to reuse the code but it will not provide fine-grain control over which domains should be excluded from SSL validation.

Reply:
Removed the comment and unused dependencies.